packages/app-store/locations.ts

```diff
@@ -359,7 +361,7 @@ export const getLocationValueForDB = (
       bookingLocation = location[eventLocationType.defaultValueVariable] || bookingLocation;
     }
   });
-  return bookingLocation;
+  return { bookingLocation, conferenceCredentialId };
```

Comment:
We need to query the event type for the location information and if a credentialId was saved